src/target/source/interface_rust.cc

```diff
@@ -0,0 +1,381 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file interface_rust.cc
+ * \brief Generates a Rust interface header for a given modules inputs and outputs
+ * which works on top of the C interface API
```

Comment:
Please come up with a proposal for resolving the goal of managing conflicted interface conventions. Specifically, given that there are multiple rust interfaces available in the TVM codebase, being able to do so would help to give space for general community.

Reply:
One possible minimum approach would be renaming the file to interface_embedded_rust.cc, or move to a subfolder that signals such intent, feel free to rename interface_c.cc together if there is an urge to keep things consistent